### ISSUE TITLE
mutlipart/form-data and Chunked transfer encoding

### ISSUE THREAD

**Issue by ufobat** [supernovus/perl6-http-easy#30]
Hi,

I am currently doing stuff with file uploads as multipart/form-data and i guess the ascii encode blows up because you can not be sure that you dont have binary data from the request inside the $first-chunk.

https://github.com/supernovus/perl6-http-easy/blob/master/lib/HTTP/Easy.pm6#L103

Chunked transfer encoding is also not supported, right?

**Comment by supernovus** [comment]
Yeah, I'm pretty sure chunked transfer encoding was on the TODO list. I haven't actively worked on my Perl 6 libraries in quite some time. Most of the work on them in the last couple of years has been contributions from others.
